determine-namespace.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e
+
+function extractStoryIdFromBranchName {
+    storyIdRegex='^(refs/heads/feature/)([0-9]{4,5})[^0-9]?(.*)$'
+
+    if [[ "$1" =~ $storyIdRegex ]]; then
+      storyId=$(echo ${BASH_REMATCH[2]}) # get the 2nd captured group
+    else
+        >&2 echo "Couldn't extract the story Id from branch name, exiting."
+        exit 1
+    fi
+}
+
+function calculateBranchNameFromBranchOfRemoteTrigger {
+  # Gets name of the variable following given pattern
+  branchNameVariableName=$(compgen -A variable | grep 'RESOURCES_PIPELINE_.*SOURCEBRANCH')
+
+  # returns the value from the variable name captured above
+  echo ${!branchNameVariableName}
+  return 0
+}
+
+function calculateNamespaceFromBranchName {
+  branchName=$1
+
+  extractStoryIdFromBranchName $branchName
+
+  allNamespaces=$(kubectl get namespaces | awk 'NR>1{print $1}')
```

Comment:
kubectl can do the work with `kubectl get ns -o=custom-columns=NAME:.metadata.name`

Reply:
That's neat, I'd prefer `kubectl` to do the work for me 😄